eslint.config.js

```diff
@@ -0,0 +1,54 @@
+import eslintConfigEslint from "eslint-config-eslint";
+import nodeRecommendedModule from "eslint-plugin-n/configs/recommended-module.js";
+import nodeRecommendedScript from "eslint-plugin-n/configs/recommended-script.js";
+import globals from "globals";
+
+export default [
+    {
+        ignores: [
+            "tests/fixtures/",
+            "dist/",
+            "coverage/",
+            "tools/create-test-example.js"
+        ]
+    },
+
+    // eslintConfigEslint[3] is eslint-plugin-n's recommended-script config
+    ...eslintConfigEslint.slice(0, 3),
+    {
+        files: ["**/*.js"],
+        ...nodeRecommendedModule
+    },
+    {
+        files: ["**/*.cjs"],
+        ...nodeRecommendedScript
+    },
+    ...eslintConfigEslint.slice(4),
+
+    {
+        settings: {
+            jsdoc: {
+                mode: "typescript"
+            }
+        }
+    },
```

Comment:
I've made a PR to move it to `eslint-config-eslint`.

Reply:
👍 

We'll remove it from here when we release eslint-config-eslint v9.